services/src/main/java/org/fao/geonet/services/metadata/format/Format.java

```diff
@@ -214,6 +217,12 @@ public void execXml(
             metadata = getXmlFromUrl(context, lang, url, request);
         }
         Element metadataEl = Xml.loadString(metadata, false);
+
+        if(mdPath != null) {
+            final List<Namespace> namespaces = context.getBean(SchemaManager.class).getSchema(schema).getNamespaces();
+            metadataEl = Xml.selectElement(metadataEl, mdPath, namespaces);
```

Comment:
What happens if the schema does not exist.  It would be better to provide a useful error message rather than a Nullpointer as you would get here.